packages/compass-import-export/src/export/export-csv.ts

```diff
@@ -223,7 +230,7 @@ async function loadEJSONFileAndColumns({
   // while simultaneously determining the unique set of columns in the order
   // we'll have to write to the file.
   const inputStream = cursor.stream();
-  const filename = temp.path({ suffix: '.jsonl' });
+  const filename = generateTempFilename('.jsonl');
```

Comment:
This package depends on `temp` already, why not just use that?

Reply:
it brings 1000 dependencies and has a vulnerability